doc/src/site/sphinx/ml/binary_model.rst

```diff
@@ -0,0 +1,142 @@
+Using H2O Binary Model in Sparkling Water
+-----------------------------------------
+
+Sparkling Water can generate binary models and can also load already existing
+models trained for example in H2O-3.
+
+Train Model in Sparkling Water and Obtain Binary model
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+We first train a model using Sparkling Water Api from which we can extract the binary model class.
+The binary model class contains methods used to work with binary models.
+
+.. content-tabs::
+
+    .. tab-container:: Scala
+        :title: Scala
+
+        .. code:: scala
+
+            import ai.h2o.sparkling._
+            val hc = H2OContext.getOrCreate()
+
+        Parse the data using H2O and convert them to Spark Frame
+
+        .. code:: scala
+
+            import org.apache.spark.SparkFiles
+            spark.sparkContext.addFile("https://raw.githubusercontent.com/h2oai/sparkling-water/master/examples/smalldata/prostate/prostate.csv")
+            val sparkDF = spark.read.option("header", "true").option("inferSchema", "true").csv(SparkFiles.get("prostate.csv"))
+
+        Train the model.
+
+        .. code:: scala
+
+            import ai.h2o.sparkling.ml.algos.H2OXGBoostClassifier
+            val estimator = new H2OXGBoostClassifier().setLabelCol("CAPSULE")
+            val mojoModel = estimator.fit(sparkDF)
+
+    .. tab-container:: Python
+        :title: Python
+
+        .. code:: python
+
+            from pysparkling import *
+            hc = H2OContext.getOrCreate()
+
+        Parse the data using H2O and convert them to Spark Frame
+
+        .. code:: python
+
+            import h2o
+            frame = h2o.import_file("https://raw.githubusercontent.com/h2oai/sparkling-water/master/examples/smalldata/prostate/prostate.csv")
+            sparkDF = hc.asSparkFrame(frame)
+
+        Train the model.
+
+        .. code:: python
+
+            from pysparkling.ml import H2OXGBoostClassifier
+            estimator = H2OXGBoost(labelCol = "CAPSULE")
+            mojoModel = estimator.fit(sparkDF)
+
+To obtain binary model once the model training has finished, run:
+
+.. content-tabs::
+
+    .. tab-container:: Scala
+        :title: Scala
+
+        .. code:: scala
+
+            estimator.getBinaryModel()
+
+    .. tab-container:: Python
+        :title: Python
+
+        .. code:: python
+
+            estimator.getBinaryModel()
+
+You can store the model on disk as:
+
+.. content-tabs::
+
+    .. tab-container:: Scala
+        :title: Scala
+
+        .. code:: scala
+
+            val binaryModel = estimator.getBinaryModel()
+            binaryModel.write("/tmp/binary.model")
+
+    .. tab-container:: Python
+        :title: Python
+
+        .. code:: python
+
+            binaryModel = estimator.getBinaryModel()
+            binaryModel.write("/tmp/binary.model")
```

Comment:
nit: WDYT are good naming pairs for such methods? `write`/`read` or `save`/`load`?

Reply:
Good catch, since we changed the createFromBinary to `load` I would change `write` to `save`